build/device.mk

```diff
@@ -74,18 +74,6 @@ PRODUCT_PROPERTY_OVERRIDES += \
     ro.radio.noril=true \
     ro.carrier=unknown
 
-# Define audio tracks that we want to use as default.
-# Do not change method of assignment because new values have to be prepended
-# and not appended (do not use +=). Explanation from google's groups:
-# "For PRODUCT_PROPERTY_OVERRIDES ... the first instance takes precedence."
-# For same reason this override has to be used after include of car.mk
-# in order to use our values.
-PRODUCT_PROPERTY_OVERRIDES := \
-     ro.config.ringtone=Atria.ogg \
```

Comment:
In car.mk we have `ringtone=Girtab.ogg`.
And we are using AudioPackage13/mk, which doesn't copy Girtab.ogg on device.
This is the reason to use our own `PRODUCT_PROPERTY_OVERRIDES` to specify files provided by AudioPackage13.

Reply:
Ok, so then we need to try PRODUCT_PROPERTY_OVERRIDES += or rethink how to solve it. In the future version, the build system will  throw error on double := to PRODUCT_PROPERTY_OVERRIDES.

Reply:
Unfortunately, we can use append (+=) due to Google's limitation "For PRODUCT_PROPERTY_OVERRIDES ... the first instance takes precedence". So if we want our change to work - we need to put our setting before same change in car.mk (as it is in discussed code).
But if this change is not allowed in next version of build system we need to change code in car.mk